app/scripts/modules/core/src/pipeline/manualExecution/PipelineOptions.tsx

```diff
@@ -106,14 +106,14 @@ export class PipelineOptions extends React.Component<IPipelineOptionsProps, IPip
       return (
         <div className="form-group row">
           <FormField
-            label={'Pipeline'}
+            label="Pipeline"
             onChange={this.pipelineSelected}
             value={formik.values.pipeline ? formik.values.pipeline.id : ''}
             input={props => (
-              <Select
+              <TetheredSelect
```

Comment:
I switched this because when you're picking a pipeline, there's only one field and the modal is super short (which makes the modal + the dropdown itself scrollable and gets awkward).